plugins/circuitbreaker/plugin_test.go

```
@@ -0,0 +1,438 @@
+		PermittedNumberOfCallsInHalfOpenState: 2,
+		MaxWaitDurationInHalfOpenState:        100 * time.Millisecond, // Short for testing
+	}
+
+	cb, err := NewCircuitBreakerPlugin(config)
+	if err != nil {
+		t.Fatalf("Failed to create circuit breaker: %v", err)
+	}
+	provider := schemas.Ollama
+
+	// Clean up any existing state
+	cb.Cleanup()
```

golangci-lint (1.64.8)

Error return value of `cb.Cleanup` is not checked

(errcheck)